utils/format.go

```diff
@@ -1141,9 +1141,9 @@ func FormatTokenBalance(balance *types.Eth1AddressBalance) template.HTML {
 	}
 	symbolTitle := FormatTokenSymbolTitle(balance.Metadata.Symbol)
 	symbol := FormatTokenSymbol(balance.Metadata.Symbol)
-	pflt, _ := price.Float64()
-	flt, _ := num.Div(mul).Round(5).Float64()
-	bflt, _ := price.Mul(num.Div(mul)).Float64()
+	pflt := price.InexactFloat64()
+	flt := num.Div(mul).Round(5).InexactFloat64()
+	bflt := price.Mul(num.Div(mul)).InexactFloat64()
```

Comment:
Not related to the BIDS but `InexactFloat64` is the same as `Float64` if you throw away the second return value anyway.